diff --git backends/virtuoso/CMakeLists.txt b/backends/virtuoso/CMakeLists.txt
index 4fd98e5..63b2134 100644
--- backends/virtuoso/CMakeLists.txt
+++ backends/virtuoso/CMakeLists.txt
@@ -34,6 +34,7 @@ if(WIN32)
   target_link_libraries(soprano_virtuosobackend odbc32)
 else()
   target_link_libraries(soprano_virtuosobackend ${IODBC_LIBRARIES})
+  SET_TARGET_PROPERTIES(soprano_virtuosobackend PROPERTIES INSTALL_RPATH "/usr/lib/odbc")
 endif()
 
 install(TARGETS soprano_virtuosobackend ${PLUGIN_INSTALL_DIR})
diff --git cmake/modules/FindIODBC.cmake b/cmake/modules/FindIODBC.cmake
index 97cb936..2e3c10f 100644
--- cmake/modules/FindIODBC.cmake
+++ cmake/modules/FindIODBC.cmake
@@ -63,10 +63,11 @@ find_path(IODBC_INCLUDE_DIR sql.h
   ${iodbc_INCLUDE_DIRS}
   )
 
-find_library(IODBC_LIBRARIES NAMES iodbc
-  HINTS
-  ${iodbc_LIBRARY_DIRS}
-  )
+#find_library(IODBC_LIBRARIES NAMES iodbc
+#  HINTS
+#  ${iodbc_LIBRARY_DIRS}
+#  )
+set(IODBC_LIBRARIES "/usr/lib/odbc/virtodbc_r.so")
 
 include(FindPackageHandleStandardArgs)
 
