Index: qwtconfig.pri
===================================================================
--- qwt/qwtconfig.pri	(revision 2391)
+++ qwt/qwtconfig.pri	(working copy)
@@ -16,10 +16,10 @@
 # Install paths
 ######################################################################
 
-QWT_INSTALL_PREFIX = $$[QT_INSTALL_PREFIX]
+QWT_INSTALL_PREFIX = /srv/jenkins/install/ubuntu/x86_64/g++/latest-qt4/general/qwt/inst/usr
 
 unix {
-    QWT_INSTALL_PREFIX    = /usr/local/qwt-$$QWT_VERSION-svn
+    # QWT_INSTALL_PREFIX    = /usr/local/qwt-$$QWT_VERSION-svn
     # QWT_INSTALL_PREFIX = /usr/local/qwt-$$QWT_VERSION-svn-qt-$$QT_VERSION
 }
 
