--- apps/haartraining/CMakeLists.txt
+++ apps/haartraining/CMakeLists.txt
@@ -14,7 +14,7 @@ if(WIN32)
   link_directories(${CMAKE_CURRENT_BINARY_DIR})
 endif()
 
-link_libraries(${OPENCV_HAARTRAINING_DEPS} opencv_haartraining_engine)
+link_libraries(${OPENCV_HAARTRAINING_DEPS})
 
 # -----------------------------------------------------------
 #  Library
@@ -39,6 +39,8 @@ set_target_properties(opencv_haartraining_engine PROPERTIES
                       INSTALL_NAME_DIR lib
                      )
 
+link_libraries(opencv_haartraining_engine)
+
 # -----------------------------------------------------------
 #  haartraining
 # -----------------------------------------------------------
