Index: cmake/modules/FindIODBC.cmake
===================================================================
--- cmake/modules/FindIODBC.cmake
+++ cmake/modules/FindIODBC.cmake
@@ -57,10 +57,11 @@
   ${iodbc_INCLUDE_DIRS}
   )
 
-find_library(IODBC_LIBRARIES NAMES iodbc
-  HINTS
-  ${iodbc_LIBRARY_DIRS}
-  )
+#find_library(IODBC_LIBRARIES NAMES iodbc
+#  HINTS
+#  ${iodbc_LIBRARY_DIRS}
+#  )
+set(IODBC_LIBRARIES "/usr/lib/odbc/virtodbc_r.so")
 
 if (IODBC_LIBRARIES AND IODBC_INCLUDE_DIR)
 #  set(IODBC_INCLUDE_DIR "${IODBC_INCLUDE_DIR}/iodbc")
Index: backends/virtuoso/CMakeLists.txt
===================================================================
--- backends/virtuoso/CMakeLists.txt
+++ backends/virtuoso/CMakeLists.txt
@@ -37,6 +37,7 @@
 target_link_libraries(soprano_virtuosobackend odbc32)
 else(WIN32)
 target_link_libraries(soprano_virtuosobackend ${IODBC_LIBRARIES})
+SET_TARGET_PROPERTIES(soprano_virtuosobackend PROPERTIES INSTALL_RPATH "/usr/lib/odbc")    
 endif(WIN32)
 
 if(WIN32)


