diff --git a/src/src.pro b/src/src.pro
index aac69b2..d23435d 100644
--- src/src.pro
+++ src/src.pro
@@ -80,11 +80,11 @@ else:unix {
     system( ./pcfile.sh $${INSTALL_PREFIX} $${VERSION} )
     pkgconfig.files = qoauth.pc
     
-    contains(QMAKE_HOST.arch, x86_64) {
-      target.path = $${INSTALL_PREFIX}/lib64
-    } else {
-      target.path = $${INSTALL_PREFIX}/lib
-    }
+#    contains(QMAKE_HOST.arch, x86_64) {
+#      target.path = $${INSTALL_PREFIX}/lib64
+#    } else {
+    target.path = $${INSTALL_PREFIX}/lib
+#    }
 
     headers.path = $${INSTALL_PREFIX}/include/QtOAuth
     docs.path = $${INSTALL_PREFIX}/share/doc/$${TARGET}-$${VERSION}/html
